src/utils/workspace-config/Cargo.toml

```diff
@@ -51,6 +53,11 @@ vendored-openssl-sys = { package = "openssl-sys", version = "0.9.96", optional =
 dynamic-zstd-sys = { package = "zstd-sys", version = "2", optional = true, default-features = false, features = [
     "pkg-config",
 ] }
+
+# FIPS
+aws-lc-rs ={ version = "1.6", optional = true, default-features = false, features = [
+    "fips",
+] }
```

Comment:
Why not enable it by default?

Reply:
I don't have a strong reason against doing so. It's simply because I didn't see any other product enabling this option by default.

Reply:
+1 for trying enable it by default.It does not bring us any additional cost I think.